types/jsonwebtoken/index.d.ts

```diff
@@ -74,7 +74,7 @@ export interface DecodeOptions {
 }
 
 export interface VerifyCallback {
-    (err: JsonWebTokenError | TokenExpiredError | NotBeforeError, decoded: any): void;
+    (err: JsonWebTokenError | NotBeforeError | TokenExpiredError, decoded: Object | string): void;
```

Comment:
@andy-ms

I did use object in the initial pull request, but changed it to Object to make Travis CI happy with older TypeScript versions.

Do you prefer object with a TypeScript Version in the definition header?

What is the prefered approach for pull requests? Use the latest and greatest features in the definition files that TypeScript has to offer or have as much backwards compatibility as possible even at the cost of being less strict?